seatunnel-core/seatunnel-starter/src/main/java/org/apache/seatunnel/core/starter/seatunnel/command/ClientExecuteCommand.java

```diff
@@ -176,4 +179,9 @@ private String creatRandomClusterName(String namePrefix) {
         return namePrefix + "-" + random.nextInt(1000000);
     }
 
+    private void shutdownHook(ClientJobProxy clientJobProxy) {
+        log.warn("Task will be closed due to client shutdown.");
```

Comment:
If job already return status to client, mean job already done, seem like we shouldn't execute cancel job.